notebooks/api_examples/migrating-to-new-api.ipynb

```diff
@@ -0,0 +1,159 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To summarise the main change:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shap_values = explainer.shap_values(X)  # Old style\n",
+    "explanation = explainer(X)  # New style"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Old style\n",
+    "\n",
+    "In versions of shap before `v0.36.0`, explanations are represented as simple numpy arrays and\n",
+    "calculated using the `.shap_values()` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[-0.54854601,  0.01639348, -0.46476041,  0.85896822, -1.36168788,\n",
+       "        -0.64692199,  0.0254638 , -0.58422904, -0.02344483,  0.        ,\n",
+       "         0.1224989 ,  0.01079906],\n",
+       "       [-0.83802091,  0.01562196,  0.78349799, -1.10456323, -0.68524691,\n",
+       "        -0.84828204,  0.03734176, -0.86151311, -0.02564897,  0.        ,\n",
+       "        -0.56183428,  0.00415988]])"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "shap_values = explainer.shap_values(X)\n",
+    "shap_values[:2]  # a numpy array"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## New style\n",
+    "\n",
+    "As of shap `v0.36.0`, explanations are represented with the `Explanation` object,\n",
+    "and are created by calling the explainer directly:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       ".values =\n",
+       "array([[-0.54854601,  0.01639348, -0.46476041,  0.85896822, -1.36168788,\n",
+       "        -0.64692199,  0.0254638 , -0.58422904, -0.02344483,  0.        ,\n",
+       "         0.1224989 ,  0.01079906],\n",
+       "       [-0.83802091,  0.01562196,  0.78349799, -1.10456323, -0.68524691,\n",
+       "        -0.84828204,  0.03734176, -0.86151311, -0.02564897,  0.        ,\n",
+       "        -0.56183428,  0.00415988]])\n",
+       "\n",
+       ".base_values =\n",
+       "array([-2.70354599, -2.70354599])\n",
+       "\n",
+       ".data =\n",
+       "array([[27.,  4., 10.,  0.,  1.,  1.,  4.,  0.,  0.,  0., 44., 39.],\n",
+       "       [27.,  4., 13.,  4., 10.,  0.,  4.,  0.,  0.,  0., 40., 39.]])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "explanation = explainer(X)\n",
+    "explanation[:2]  # a shap.Explanation object"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `Explanation` object is a much richer representation that includes the shap values\n",
```

Comment:
I would add a cell where it is explained that using `explanation.values[:2]` yields the same result as the calculation using the old style.

Reply:
What is the purpose of the `[:2]` indexing, is the shape different?

Reply:
nope, it is just done for the .shap_values call as well. One could then make an equal vomparison or just state that the vlaues are equal

Reply:
Done 👍